ember/ember-cli-build.js

```diff
@@ -1,47 +1,13 @@
 "use strict";
 
-//eslint-disable-next-line node/no-missing-require, node/no-unpublished-require
 const EmberApp = require("ember-cli/lib/broccoli/ember-app");
 
 module.exports = function (defaults) {
   const app = new EmberApp(defaults, {
-    sourcemaps: { enabled: true },
-    "ember-cli-babel": {
-      includePolyfill: true,
-    },
-
-    sassOptions: {
-      includePaths: [
-        // For some reason sass cant find these on its own :/
-        "node_modules/ember-power-select/app/styles/",
-        "node_modules/ember-basic-dropdown/app/styles/",
-      ],
```

Comment:
They are included now?

Reply:
@derrabauke  i don't see any error, and i don't see something wrong in the UI.
```scss
@import "ember-alexandria";
@import "ember-emeis";
@import "ember-power-select";
@import "ember-uikit";
@import "@projectcaluma/ember-form";
@import "@projectcaluma/ember-form-builder";
```
but in `app.scss` file, you can see that this is the new way of importing the styles